### PR TITLE
DYN-3836: [BASH] Buttons looks strange when not hovered

### PR DESCRIPTION
**Pull request by sm6srw**
### Purpose

This pull request does:

* add a new solid button style for use in preferences.

image

image

I have followed the style we where pointed at but the buttons feels a little too large for me. What do you think? @Jingyi-Wen

### Declarations

Check these if you believe they are true

- [X] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [X] All tests pass using the self-service CI.
- [X] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated

### Release Notes

(FILL ME IN) Brief description of the fix / enhancement. **Mandatory section**

### CODE REVIEW

**Comment by mjkkirschner** [comment]
@sm6srw would you mind adding a screenshot?

File: src/DynamoCoreWpf/UI/Themes/Modern/DynamoModern.xaml [hunk 1]
```diff
@@ -2582,6 +2582,62 @@
         </Setter>
     </Style>
 
+    <Style x:Key="SolidButtonStyle" TargetType="Button">
```
[the remark this answers is not in the document]
Comment (sm6srw) [reply src/DynamoCoreWpf/UI/Themes/Modern/DynamoModern.xaml:2585]: There is no style for this yet that works on a gray background. There is one that works on a white background (e.g. the new message boxes). I tried to combine them but it was always something that didn't work. So this is the best I can do for now I think.

**Comment by sm6srw** [comment]
> @sm6srw would you mind adding a screenshot?

Sorry forgot, added now.

**Comment by sm6srw** [comment]
Approved by UX. PTAL!

**Review by mjkkirschner — APPROVED** [review]

**Comment by OliverEGreen** [comment]
@sm6srw FYI: this looks a lot like the `CtaButtonStyle` in `DynamoModern.xaml`

**Pull request merged**